build.gradle

```diff
@@ -12,7 +12,7 @@ plugins {
 
 
 ext.versions = [
-    checkerFramework: "3.8.0",
+    checkerFramework: "3.9.2-SNAPSHOT",
```

Comment:
Make this `3.10.0` I think

Reply:
https://github.com/kelloggm/object-construction-checker/pull/291/commits/790ac07e99314570d32d4c8b2fc03158226ec15a